src/js/components/FileInput/FileInput.js

```diff
@@ -25,7 +25,7 @@ const ContentsBox = styled(Box)`
   position: relative;
   ${props => (props.disabled ? disabledStyle() : 'cursor: pointer;')}
   &:focus {
-    ${focusStyle}
+    online: none;
```

Comment:
what does "online" mean?

Reply:
opps lol not sure how that got there suppose to be `outline`